custom_components/teslafi/sensor.py

```diff
@@ -246,6 +246,26 @@
         available=lambda u, d, h: u and d.tpms.rear_right,
     ),
     # endregion
+    # region TeslaFi API Counts
+        TeslaFiSensorEntityDescription(
+        key="commands",
+        name="API Commands",
+        icon="mdi:counter",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=True,
+        value=lambda d, h: d.api_requests.commands,
+        available=lambda u, d, h: u and d.api_requests.commands,
```

Comment:
And same here: the parent class will already handle this, generally speaking. 

These lambdas exist only to give us a way to do _non-default_ behaviors.

Look at `outside_temp` for example. It just sets the dictionary key and some metadata like units. The parent class handles the rest